install.rb

```diff
@@ -410,7 +485,7 @@ def install_binfile(from, op_file, target)
 # Change directory into the puppet root so we don't get the wrong files for install.
 FileUtils.cd File.dirname(__FILE__) do
   # Set these values to what you want installed.
-  configs = glob(%w{conf/auth.conf})
+  configs = glob(%w{conf/auth.conf conf/puppet.conf})
```

Comment:
By adding `puppet.conf` to Windows, we get errant paths injected into the environment for `vardir`, `logdir` and `rundir`.  As soon as `puppet.conf` is gone, things are back to normal.

See https://github.com/puppetlabs/puppet/pull/3556#issuecomment-73300673